LibreNMS/OS/Traits/BridgeMib.php

```diff
@@ -50,7 +50,7 @@ public function discoverStpInstances(): Collection
         foreach ($vlans->isEmpty() ? [null] : $vlans as $vlan) {
             // fetch STP config and store it
             $vlan = (empty($vlan->vlan_vlan) || $vlan->vlan_vlan == '1') ? null : $vlan->vlan_vlan;
-            $instance = SnmpQuery::context("$vlan", 'vlan-')->enumStrings()->get([
+            $instance = SnmpQuery::context("$instance->vlan", 'vlan-')->enumStrings()->get([
```

Comment:
Hi @erotel 
I am not sure I get it here. You define `$instance` using one of its properties ? 
Thx

Reply:
i don't know if it's ok, i'm not a php programmer. i just tried it on my librenms and it works. i tested it on huawei, edge-core, dcn and zyxel switches. if it's nonsense then delete that pull